README.md

```diff
@@ -87,11 +88,12 @@ cbioportal_etl \
 
 ### Steps Argument
 The `--steps` argument specifies which steps of the pipeline to run. It is outlined as follows:
-- `1` - Get study metadata
-- `2` - Compare current DWH data vs cBioPortal build
-- `3` - Get files from manifest
-- `4` - Check downloaded files
-- `5` - Build genomic file package
+- `1` - Generate config JSON
```

Comment:
instead of bullet point plus manually enter number, you can just put `1.` in front of each entry, and markdown will automatically treat that as a list and increment for you